### PR TITLE
Create classic-taskbar-buttons-lite.wh.cpp

### PR DESCRIPTION
**Pull request by Anixx**
3D taskbar buttons for Classic theme, lightweight

### CODE REVIEW

**Comment by m417z** [comment]
Please include `@github`.

**Comment by Anixx** [comment]
Done

File: mods/classic-taskbar-buttons-lite.wh.cpp [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+#include <windhawk_utils.h>
+
+
+typedef struct tagBUTTONRENDERINFOSTATES {
+    char data[12];
+} BUTTONRENDERINFOSTATES, *PBUTTONRENDERINFOSTATES;
+
+/* Draw taskbar item */
+typedef void (* CTaskBtnGroup__DrawBar_t)(void *, HDC, void *, void *);
```
Comment (m417z) [inline mods/classic-taskbar-buttons-lite.wh.cpp:44]: Calling conventions are missing, which is likely to cause a crash for 32-bit targets. Please add calling conventions or, if the mod doesn't support 32-bit, limit it to 64-bit, which will also make compilation faster. See also:
https://github.com/ramensoftware/windhawk/wiki/Development-tips#hooks-and-calling-conventions

**Comment by Anixx** [comment]
Done. Not sure if I did it right though. Do you mind including these declarations into Windhawk includes or a mod template?

**Comment by Anixx** [comment]
Wait, it seems, it now does not work... I have no idea...

**Comment by Anixx** [comment]
Fixed now.

**Pull request merged**